m_flat_generator.py

```diff
@@ -0,0 +1,51 @@
+from astropy import units as u
+import os
+from pathlib import Path
+from astropy.nddata import CCDData
+import numpy as np
+from astropy.stats import mad_std
+import ccdproc as ccdp
+from astropy.io import fits
+
+
+def inv_median(a):
+    return  1/np.median(a)
+main_p = '2023_01_05'
+flat_p = '2023_01_05/flat'
+flat_image_type = 'flat'
+m_dark_p = 'master_dark.fits'
+m_bias_p = 'master_bias.fits'
+ifc_flat = ccdp.ImageFileCollection(flat_p)
+print("LOG:zaladowano flaty:",ifc_flat)
+master_bias  = CCDData(fits.getdata(m_bias_p), meta=fits.getheader(m_bias_p),unit = 'adu')
+print("LOG:otworzono m_biasa",m_bias_p)
+master_dark  = CCDData(fits.getdata(m_dark_p), meta=fits.getheader(m_dark_p),unit = 'adu')
+print("LOG:otworzono m_darka",m_dark_p)
+master_dark = ccdp.subtract_bias(master_dark, master_bias)
+print("LOG:bias usuniety z master_darka")
+flat_filters = set(h['filter'] for h in ifc_flat.headers(imagetyp=flat_image_type))
+
+dark_expo = master_dark.header['EXPTIME']
+for filt in flat_filters:
+    flat_list = ifc_flat.files_filtered(imagetyp = flat_image_type, filter = filt,include_path=True)
+    print("LOG:W filtrze",filt, "mamy tyle flatow", len(flat_list))
+    CCDDATA_flat_list = []
```

Comment:
konwencja nazw zmiennych sugeruje uzycie "lower-case" wiec
```suggestion
    cccdata_flat_list = []
```